scripts/zones/Metalworks/npcs/Ayame.lua

```diff
@@ -28,13 +57,16 @@ function onTrigger(player,npc)
 
     local trueStrength = player:getQuestStatus(BASTOK,tpz.quest.id.bastok.TRUE_STRENGTH);
     local WildcatBastok = player:getCharVar("WildcatBastok");
+    local Rank3 = player:getRank(player:getNation()) >= 3 and 1 or 0
```

Comment:
I realize I originally pointed out `getNation()` when I saw the individual checks before, but a thought just occurred: `getRank` returns the player's current nation by default.

And upon checking, it apparently _can't_ give rank for a specific nation. [It was suggested to be added in the comments, but never was](https://github.com/project-topaz/topaz/blob/release/src/map/lua/lua_baseentity.cpp#L6003-L6016).